src/components/AppTodo.js

```diff
@@ -0,0 +1,72 @@
+import { LitElement, html, css } from 'lit-element';
+
+import './AppButton';
+
+class AppTodo extends LitElement {
+
+  static get properties() {
+    return {
+      todo: { type: Object }
+    }
+  }
+
+  static get styles() {
+    return css`
+      .todo {
+        background-color: #fff;
+        padding: 20px;
+        margin: 20px 0;
+        border-radius: 6px;
+      }
+      h2, p {
+        margin: 0;
+        padding: 0;
+      }
+      .todo.complete h2, .todo.complete p {
+        text-decoration: line-through;
+      }
+      .todo-actions {
+        margin-top: 20px;
+      }
+    `;
+  }
+
+  constructor() {
+    super();
+  }
+
+  handleDeleteTodo = () => {
+    this.deleteHandler(this.todo.id);
+  }
+
+  handleToggleActive = () => {
+    this.toggleHandler(this.todo.id);
+  }
+
+  render() {
+    return html`
+    <div class="todo ${!this.todo.active ? 'complete' : ''}">
+      <h2>${this.todo.title}</h2>
+      <p>${this.todo.description}</p>
```

Comment:
what if in an initial render, we have `this.todo` not defined yet?
This would definitely result in an error.